lib/y2logsstep.pm

```diff
@@ -57,61 +58,130 @@ sub get_to_console() {
     }
 }
 
-# to workaround dep issues
-sub record_dependency_issues {
-    while (check_screen 'dependancy-issue', 5) {
-        wait_screen_change {
-            if (check_var('VIDEOMODE', 'text')) {
-                send_key 'alt-s';
-            }
-            else {
-                send_key 'alt-1';
-            }
-        };
-        wait_screen_change {
+# to workaround dependency issues
+sub workaround_dependency_issues {
+    if (check_var('VIDEOMODE', 'text')) {
+        while (check_screen('dependency-issue-text', 5)) {
+            send_key 'alt-s';
+            send_key 'ret';
+            send_key 'alt-o';
```

Comment:
Most of the time the screen almost doesn't change at all, thus go back to `send_key`.